integration-tests/bats/replication.bats

```diff
@@ -108,6 +108,35 @@ teardown() {
     [[ ! "$output" =~ "feature" ]] || false
 }
 
+@test "replication: push tag delete, pull delete on read" {
```

Comment:
just double checking that you meant to include this

Reply:
Yeah, it was missing from my first PR

Reply:
Yeah it was missing from my first PR